src/util/articleListMainPage.ts

```diff
@@ -6,7 +6,7 @@ export function makeArticleListItem(dump: Dump, articleEntry: any) {
 }
 
 export function makeArticleImageTile(dump: Dump, articleEntry: ArticleDetail, webp: boolean) {
-    return `<a class="item" href="${'./' + encodeArticleIdForZimHtmlUrl(articleEntry.title.replace(/ /g, '_') || '')}">
+    return `<a class="item" href="${encodeArticleIdForZimHtmlUrl('./' + articleEntry.title.replace(/ /g, '_'))}">
```

Comment:
@kelson42 the reason for changing this line is that if `articleEntry.title` starts with a `/` it will convert it to start with `./` and then as we were already adding one `./` to it after `encodeArticleIdForZimHtmlUrl()` was called we would end up having `././`. I guess that won't cause any problem but, still would have been wrong and in case `title` was empty we would just have `./` as the href, so what I have done now does the same thing and just removes the problem of two consecutive  `./` in href